### ISSUE TITLE
DeprecationWarning

### ISSUE THREAD

**Issue by impguard** [ItsNickBarry/hardhat-abi-exporter#19]
Seeing this warning on the latest LTS version of node (16.13.0):

```
(node:21035) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
```

Should be a small fix. If possible!

**Comment by ItsNickBarry** [comment]
Same issue in the `hardhat-dependency-compiler` repository, see my comment: https://github.com/ItsNickBarry/hardhat-dependency-compiler/pull/5

**Comment by ItsNickBarry** [comment]
Fixed in version `2.3.1`.
